src/__stories__/bottom-sheet-story.tsx

```diff
@@ -0,0 +1,541 @@
+import * as React from 'react';
+import {
+    BottomSheet,
+    BottomSheetRoot,
+    Box,
+    ButtonPrimary,
+    Circle,
+    IconCheckRegular,
+    IconCocktailRegular,
+    IconLightningRegular,
+    IconMobileDeviceRegular,
+    IconTrashCanRegular,
+    Inline,
+    Placeholder,
+    ResponsiveLayout,
+    showBottomSheet,
+    skinVars,
+    Stack,
+    Text2,
+    Text3,
+    Title1,
+} from '..';
+import {
+    ActionsBottomSheet,
+    ActionsListBottomSheet,
+    InfoBottomSheet,
+    RadioListBottomSheet,
+} from '../bottom-sheet';
+import avatarImg from './images/avatar.jpg';
+
+export default {
+    title: 'Components/Modals/BottomSheet',
+    component: BottomSheet,
+    parameters: {
+        fullScreen: true,
+    },
+};
+
+export const Default: StoryComponent = () => {
+    const [open, setOpen] = React.useState(false);
+
+    return (
+        <Box paddingY={24}>
+            <ResponsiveLayout>
+                <ButtonPrimary
+                    disabled={open}
+                    onPress={() => {
+                        setOpen(true);
+                    }}
+                >
+                    Open
+                </ButtonPrimary>
+
+                {open && (
+                    <BottomSheet
+                        onClose={() => {
+                            setOpen(false);
+                        }}
+                    >
+                        <ResponsiveLayout>
+                            <Box
+                                paddingBottom={{mobile: 16, desktop: 40}}
+                                paddingTop={{mobile: 0, desktop: 40}}
+                            >
+                                <Placeholder />
+                            </Box>
+                        </ResponsiveLayout>
+                    </BottomSheet>
+                )}
+            </ResponsiveLayout>
+        </Box>
+    );
+};
+
+Default.storyName = 'BottomSheet';
```

Comment:
this type of sheet also includes `Title`, `Subtitle` and `Description` elements

Reply:
hmmm... no. This is the basic component, it can have anything inside it. You can combine it with SheetBody if you need title/subtitle/description

Reply:
Great, can you give me an example in playroom? ☺️

Reply:
something like this:
https://mistica-gseky3w6e-tuentisre.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AFATjAbgSxgdwBUIIAbAZwAIB6APgB0A7RpAIQFcAXDiB9bAWwCGaAJ6MKFKNjKCARiRhQAvMADmMDgGUOgjjAAUdEGQAWEPEYCUAX3EUe6GGTIr9likpoVgdiWQ3augZGpuZGADQUHGhsMJYA3HbWtgz0DBKaZngUpjAazFTsXA5oAsIiaYxqATp6hsZZVhQAZM0UhukUSJl5HL72DADCJBD%2Bru6e3v1%2BNUH1oRYgkQBmguRxiZ0SyXZpEhLdJr0s0CJR2BwKSkYEFwpGOWyyHHcw18ZPL5cwD7BkYKUAA4vHjvP4A7DA7A8B60OxIKg9DRpGyMJYgDhHfhOBAAbRALBIgjYIAAuuEQHhsFBMWQ8QBmABsAAZwgBGZkAJgALKTrEA